bundle/edu.gemini.model.p1/src/test/resources/edu/gemini/model/p1/immutable/proposal_itac_no_ngoauthority.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <proposal tacCategory="Extragalactic" schemaVersion="1.0.0">
     <meta>
-        <attachment name="ar_2012B_007.pdf" index="1"/>
-        <attachment name="ar_2012B_007.pdf" index="2"/>
```

Comment:
they can be the same file in the end?

Reply:
So this is just a test, and currently there is no restriction. But it is a good question whether we should verify it